utils/crypto_dir.go

```diff
@@ -115,5 +116,40 @@ func BackupKubeconfigCrypto(ctx context.Context, client runtimeclient.Client, cr
 	if err := os.WriteFile(path.Join(cryptoDir, "ingresskey-ingress-operator.key"), []byte(ingressOperatorKey), cryptoDirMode); err != nil {
 		return err
 	}
+
 	return nil
 }
+
+func BackupKubeadminPasswordHash(ctx context.Context, client runtimeclient.Client, cryptoDir string) (bool, error) {
+	password, err := GetSecretData(ctx, "kubeadmin", "kube-system", "kubeadmin", client)
+	if err != nil {
+		if runtimeclient.IgnoreNotFound(err) != nil {
```

Comment:
A "not found" error (err == nil) is a valid case and should return nil error
```suggestion
		if runtimeclient.IgnoreNotFound(err) == nil {
```

Reply:
Thanks